GOVERNANCE-subprojects.md

```diff
@@ -0,0 +1,164 @@
+* Arbitrating inter-project disagreements
+* Selecting the Code of Conduct Committee and ratifying CoC judgements
+* Removing projects which have become inactive
+* Acting on escalated requests from security or code quality issues
+* Resolving project issues that individual projects are unable to resolve
+* Administering project infrastructure, intellectual property, and resources
+* Determining overall direction for advocacy and marketing
+* Issuing statements on behalf of [TODO:PROJECTNAME] and its projects
+* Reviewing and approving Contributor Ladder advancement for participants who
+  work on the overall umbrella project
+
+In performance of these duties, the Steering Committee will hold a monthly meeting
+that is open to all Members.  The Committee may hold additional, closed meetings
+in order to discuss non-public issues such as security exploits, CoC enforcement,
+and legal questions.
+
+Steering Committee members are expected to advocate for all of [TODO:PROJECTNAME], not just
+certain projects or corporate sponsors, comply with and support the Code of
+Conduct, and be professional and compassionate in all of their dealings with
+project participants.
+
+### Steering Committee Elections
+
+The Maintainer Representative(s) will be elected by the collective Maintainers of
+all projects, as defined in the Contributor Ladder and accepted by the
+projects.  The Member Representative(s) will be elected by the collective Members
+of all projects, as defined in the Contributor Ladder.  These positions
+will be elected annually.
+
+Once per year, the Steering Committee will select between one and three Election
+Officers to run the annual election and sets the dates for the election.  These
+officers will update the list of eligible Maintainers and Members according to
+project records, send out announcements, and conduct the election.  The election
+starts with a call for nominations, which lasts for at least one week, and will
+include a period for project Members to request changes to their voting status.
+Candidates may nominate themselves, or they may be nominated by their colleagues.
+
+The election itself will last for at least one week, and is conducted as a
+preference election online, using a method such as Condorcet or IRV.  The
+Election Officers will announce the selected candidates at the next regular
+community meeting.
+
+## Code of Conduct Committee
+
+<!--omit the below if you've already defined a detailed CoC Committee elsewhere,
+or are relying entirely on the CNCF for CoC enforcement-->
+
+In order to review and enforce the Code Of Conduct, the Steering Committee selects
+[TODO:Number, suggest "3 to 5"] people to be on the Code of Conduct Committee.  These individuals
+will be chosen based on their experience in community management and conduct
+issues, and to represent a diversity of viewpoints, including employer, gender,
+race, background, and region of the world.  To avoid fatigue, the Steering Committee will
```

Comment:
This sentence seems awkward to me: Maybe:
These individuals will be chosen based on their community management and code of conduct
experience with diverse representation across the committee, including employer, gender,
race, background, and region of the world.

Reply:
thanks, replaced